server/src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -284,6 +288,23 @@
      */
     void open(OpenIndexRequest request, ActionListener<OpenIndexResponse> listener);
 
+    /**
+     * Adds a block to an index
+     *
+     * @param block   The block to add
+     * @param indices The name of the indices to add the block to
+     */
+    AddIndexBlockRequestBuilder prepareAddBlock(APIBlock block, String... indices);
```

Comment:
I don't think we need to add these since we are not supporting the transport client any longer?

Reply:
see comment above about transport client